.github/ISSUE_TEMPLATE/blank.yaml

```diff
@@ -120,6 +120,7 @@ body:
       - otelcol.processor.transform
       - otelcol.receiver.awscloudwatch
       - otelcol.receiver.awsecscontainermetrics
+      - otelcol.receiver.cloudflare
```

Comment:
Maybe more of a question for @dehaansa

Does the automation you wrote update these automatically? Or maybe that's what was used here already to include the new component in the templates?

Reply:
@jharvey10 I had to manually add this as one of CI jobs checks presence of each component.